django/db/migrations/state.py

```diff
@@ -135,6 +144,16 @@ def rename_model(self, app_label, old_name, new_name):
                 to_reload.add((model_state.app_label, model_state.name_lower))
         # Reload models related to old model before removing the old model.
         self.reload_models(to_reload, delay=True)
+        old_name_key = (app_label, old_name.lower())
+        new_name_key = (app_label, new_name.lower())
+        if (app_label, old_name.lower()) in self.relations:
+            self.relations[new_name_key] = self.relations[old_name_key]
+            self.relations.pop(old_name_key, '')
+        relations = copy.copy(self.relations)
+        for key, relation in relations.items():
+            if old_name_key in relation:
+                self.relations[key][new_name_key] = relation[old_name_key]
+                self.relations[key].pop(old_name_key, '')
```

Comment:
Use more significant variable names here and take advantage of `.pop` return value

```suggestion
        for model_relations in relations.values():
            if old_name_key in model_relations:
                model_relations[new_name_key] = model_relations.pop(old_name_key)
```